pkg/drivers/hyperv/hyperv.go

```diff
@@ -204,7 +204,7 @@ func (d *Driver) Create() error {
 	}
 
 	log.Debugf("Starting VM...")
-	return d.Start()
+	return nil
```

Comment:
Shouldn't `log.Debugf("Starting VM...")` also removed it from here and have this where we actually call the driver start?

Reply:
good catch! removing this.